ros2/test/launch_pytest/BUILD.bazel

```diff
@@ -0,0 +1,18 @@
+load("@rules_ros2_pip_deps//:requirements.bzl", "requirement")
+
+cc_binary(
+    name = "hello",
+    srcs = ["hello.cpp"],
+)
+
+py_test(
```

Comment:
The existing `ros2_test` is tied to `launch_testing`. The idea of `launch_pytest` is that you use `pytest` as the test driver. I would implement a new rule `ros2_launch_pytest` that accepts a `nodes` argument and sets up a `pytest` internally similarly to how I did it manually here.